### PR TITLE
Set up backend

### PR DESCRIPTION
**Pull request by calle1212**
Bogus seeder Endpoint. GetAllProjects Endpoint.

### CODE REVIEW

File: Backend/Models/DefaultDataType.cs
Comment (rorychatt) [inline Backend/Models/DefaultDataType.cs:1]: TRASH

**Comment by Oshulten** [comment]
Look's all good!

**Review by Oshulten — APPROVED** [review]
Nice work!

**Comment by Oshulten** [comment]
The endpoint desperately needs a dto (flat)

**Comment by Oshulten** [comment]
There is no way to differentiate between a skill tag and an interest tag

**Comment by rorychatt** [comment]
yeah shit code

**Review by timabloom — COMMENTED** [review]
Good work but we need to differentiate between skill and interest tag

**Pull request merged**